modules/deploy/pages/deployment-option/cloud/configure-privatelink-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,88 @@
+GCP::
++
+* Use https://cloud.google.com/sdk/docs/install[gcloud^] to create the consumer-side resources, such as a VPC and forwarding rule, or modify an existing one to use the Private Service Connect service attachment created for your cluster.
+====
+
+== Enable endpoint service for existing clusters
+
+[tabs]
+======
+AWS::
++
+. In the Redpanda Cloud UI, open your https://cloud.redpanda.com/clusters[cluster^], and click **Cluster settings**.
+. Under AWS PrivateLink, click **Enable**. 
+. You need the Amazon Resource Names (ARNs) for the AWS principals allowed to access the endpoint service. For example, for all principals in a specific account, use `arn:aws:iam::<account-id>:root`. See https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permission[Configure an endpoint service^] for details.
+. It may take several minutes for your cluster to update. When the update is complete, the AWS PrivateLink status in **Cluster settings** changes from **In progress** to **Enabled**.
+GCP::
++
+. In the Redpanda Cloud UI, open your https://cloud.redpanda.com/clusters[cluster^], and click **Cluster settings**.
+. Under Private Service Connect, click **Enable**. 
+. For xref:deploy:deployment-option/cloud/vpc-byo-gcp.adoc[BYOC clusters with customer-managed VPC], you need a NAT subnet with the *Purpose* set to `PRIVATE_SERVICE_CONNECT`. You can create the subnet using the `gcloud` command-line interface (CLI):
++
+[,bash]
+----
+gcloud compute networks subnets create <subnet-name> \
+    --project=<project> \
+    --network=<network-name> \
+    --region=<region> \
+    --range=<subnet-range> \
+    --purpose=PRIVATE_SERVICE_CONNECT
+----
++
+Provide your values for the following placeholders:
++
+- `<subnet-name>`: The name of the NAT subnet.
+- `<project>`: The **host** GCP project ID.
+- `<network-name>`: The name of the VPC being used for your Redpanda Cloud cluster.
+- `<region>`: The region of the Redpanda Cloud cluster.
+- `<subnet-range>`: The CIDR range of the subnet. The mask should be at least `/29`. Each Private Service Connect connection takes up one IP address from the NAT subnet, so the CIDR must be able to accommodate all projects from which connections to the service attachment will be issued.
++
+See the official GCP documentation for https://cloud.google.com/vpc/docs/configure-private-service-connect-producer#add-subnet-psc[creating a subnet for Private Service Connect^].
+. For the accepted consumers list, you need the GCP project IDs from which incoming connections will be accepted.
+. It may take several minutes for your cluster to update. When the update is complete, the Private Service Connect status in **Cluster settings** changes from **In progress** to **Enabled**.
+======
+
+== Access Redpanda services through VPC endpoint
```

Comment:
@0x5d @paulzhang97 just want to confirm whether this section is relevant and should also be included in this doc for the UI.

Reply:
Yes.